src/types/value/table.ts

```diff
@@ -0,0 +1,64 @@
+import type { FieldState } from "./types"
+
+import type { RichTextField } from "./richText"
+
+/**
+ * A table field.
+ *
+ * @typeParam State - State of the field which determines its shape.
+ *
+ * @see More details: {@link https://prismic.io/docs/table}
+ */
+export type TableField<State extends FieldState = FieldState> =
```

Comment:
This content model is out of date. It will need to be updated when you update the `types-internal` alpha to a newer version.